lib/routing/router.ts

```diff
@@ -74,9 +93,36 @@ export class Router<M extends Middleware, C extends Controller, R extends Route<
   }
 
   public async dispatchController(): Promise<Response> {
+    const getParameters = (func) => {
+      // First match everything inside the function argument parens.
+      var args = func.toString().match(/function\s.*?\(([^)]*)\)/)[1];
+     
+      // Split the arguments string into an array comma delimited.
+      return args.split(",").map(function(arg) {
+        // Ensure no inline comments are parsed and trim the whitespace.
+        return arg.replace(/\/\*.*\*\//, "").trim();
+      }).filter(function(arg) {
+        // Ensure no undefineds are added.
```

Comment:
how does this ensure no undefinds are added

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter

it filters out values when return val is falsey. need to update this all to be es6y